analyzer/util/util.go

```diff
@@ -103,3 +103,13 @@ func SanitizeTxHash(hash string) *string {
 
 	return &hash
 }
+
+func TxHashToIndex(hash string, lookup map[string]int) *int {
```

Comment:
Nit: Since this fn knows nothing about hashes or txs, can we call it `GetOrNil(key string, kvs map[string][int])` or similar?

Reply:
Removed.